content/en/agent/kubernetes/prometheus.md

```diff
@@ -74,15 +79,17 @@ With the following configuration placeholder values:
 | `<METRIC_TO_FETCH>`                      | Prometheus metrics key to be fetched from the Prometheus endpoint.                                                                                                                                             |
 | `<NEW_METRIC_NAME>`                      | Optional parameter which, if set, transforms the `<METRIC_TO_FETCH>` metric key to `<NEW_METRIC_NAME>` in Datadog. If you choose not to use this option, pass a list of strings rather than `key:value` pairs. |
 
-**Note**: See the [sample openmetrics.d/conf.yaml][8] for all available configuration options.
+**Note**: See the [sample openmetrics.d/conf.yaml][9] for all available configuration options.
+
+For a full list of available parameters for instances, including `namespace` and `metrics`, see the table in the [Prometheus host collection guide][10].
```

Comment:
I'd suggest to put the "For a full list of..." line above the "Note". 

Additionally, I'd also propose to remove the "Note" completely from this Kubernetes page. It appears in the "Docker" page where conf.yaml files are used, but they are rarely used in Kubernetes